fili-core/src/main/java/com/yahoo/bard/webservice/web/filters/BardLoggingFilter.java

```diff
@@ -83,6 +84,7 @@
     @Override
     public void filter(ContainerRequestContext request) throws IOException {
 
+        RequestLog.addIdPrefix(request.getHeaders().getFirst(X_REQUEST_ID_HEADER));
```

Comment:
Yeah, that'll work, though it would be good to add periods and underscores (`.` and `_`)